client/components/App.js

```diff
@@ -1,32 +1,61 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { HashRouter as Router, Route } from 'react-router-dom';
-import Home from './home/Home'
-import Build from './build/Build'
-import QuestionWalkthrough from './build/QuestionWalkthrough'
-import NavBar from './NavBar'
-import SignIn from './SignIn'
+import axios from 'axios';
+import Home from './home/Home';
+import Build from './build/Build';
+import QuestionWalkthrough from './build/QuestionWalkthrough';
+import NavBar from './NavBar';
+import SignIn from './SignIn';
 
-function App() {
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      user: false,
+    };
+    this.logout = this.logout.bind(this);
+  }
 
+  componentDidMount() {
+    // upon load, check if there is a loged in user
+    // in future, use cookies instead LS
+    const token = window.localStorage.getItem('token');
```

Comment:
In the interest of keeping things uniform, I might recommend using React Hooks and the UseEffect hook rather than turning this into a class component and using a constructor & componentDidMount. It's no biggie if you'd rather do it this way, but I'd be happy to walk through how to implement it if you're interested!